backend/tests/v5/unit/middleware/dataConverter/inputs/users.test.js

```diff
@@ -64,7 +65,9 @@ UsersModel.getUserByQuery.mockImplementation((query) => {
 
 UsersModel.getUserByUsernameOrEmail.mockImplementation((usernameOrEmail) => {
```

Comment:
Can we refactor this to do mockImplementationOnce/mockResolvedValueOnce within the test?